workflows/ci_failure_summarizer/summarize.py

```
@@ -0,0 +1,252 @@
+    
+    response = requests.post(
+        f"{OLLAMA_URL}/api/chat",
+        json=payload,
+        timeout=120  # 2 min timeout for inference
+    )
+    response.raise_for_status()
+    
+    result = response.json()
+    
+    if "message" not in result or "content" not in result["message"]:
+        raise ValueError(f"Unexpected Ollama response format: {result}")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)